makefile

```diff
@@ -11,9 +10,9 @@ lint:
 
 .PHONY: format
 format:
-ifneq ("$(FILES)"," ")
-	black $(FILES)
-	isort $(FILES)
+ifneq ("$(FILES)","")
+	black $(FILES) $(args)
+	isort $(FILES) --line-length 100 --profile black
```

Comment:
isort's configured in pyproject.toml
What's "$(args)" for?

Reply:
ah - I didn't mean to leave this in. I was getting some makefile errors and was looking at what `sync_backend` was doing to try and resolve them

Reply:
The issue was the `$(FILES)` is equal to `""`, not `" "`, when there are no `.py` files in the git diff, which was causing some errors from `black`.

Reply:
Nice bug fix

Reply:
Probably want to include this in `sync_backend` as well, since I think the check there is the same...